README.md

```diff
@@ -167,7 +167,7 @@ Happy querying!
 - `engine (Engine)`: a Tartiflette [engine](https://tartiflette.io/docs/api/engine). Required if `sdl` is not given.
 - `sdl (str)`: a GraphQL schema defined using the [GraphQL Schema Definition Language](https://graphql.org/learn/schema/). Required if `engine` is not given.
 - `graphiql (bool)`: whether to serve the GraphiQL when accessing the endpoint via a web browser. Defaults to `True`.
-- `path (str)`: the path which clients should make GraphQL queries to. Defaults to `"/"`. A popular alternative is `"/graphql"`.
+- `path (str)`: the path which clients should make GraphQL queries to. Defaults to `""`.
```

Comment:
Is it possible to expose the `request` object into the "req" key of `context`? As `tartiflette-aiohttp` do?

And the last thing, expose the application object (if possible), into the `app` key of `context`?

e.g. tartiflette-starlette
```python
@Resolver("Query.whoami")
async def resolve_whoami(parent, args, context, info) -> str:
    request = context["request"]
    return getattr(request.state, "user", "a mystery")
```

e.g tartilfette-aiohttp
```
executor_context: Context which will be passed to each resolver (as a dict). Very useful for passing handlers to services, functions or data that you want to use in your resolvers. The context reference is unique per request, a shallow copy is created based on the context passed.
    req: Request object from aiohttp
    app: Application object from aiohttp
```

Reply:
Tracked in #24